doc/dev/decisionrecords/Codestyle.md

```diff
@@ -1,96 +1,96 @@
 
 The OpenTripPlanner Java code style is revised in OTP v2.2. We use the
-[Prettier Java](https://github.com/jhipster/prettier-java) as is. Maven is setup to
+[Prettier Java](https://github.com/jhipster/prettier-java) as is. Maven is set up to
 run `prettier-maven-plugin`. A check is run in the CI build, which fails the build preventing
-merging a PR if the code-style is incorrect.
+merging a PR if the code style is incorrect.
 
-There is two ways to format the code before checkin it in. You may run a normal build with maven -
-it takes a bit of time, but reformat the entire codebase. Only code you have changed should be
-formatted, since the existing code is already formatted. The second way is to set up prettier and
+There are two ways to format the code before checking it in. You may run a normal build with
+Maven—it takes a bit of time, but reformat the entire codebase. Only code you have changed should be
+formatted, since the existing code is already formatted. The second way is to set up Prettier and
 run it manually or hick it into your IDE, so it runs every time a file is changed.
 
-### How to run Prettier with Maven
+### How to Run Prettier with Maven
 
-Prettier will automatically format all code in the Maven "validate" phase, which runs before the test, package, and install phases. So formatting will happen for example when you run:
+Prettier will automatically format all code in the Maven "validate" phase, which runs before the
+test, package, and install phases. So formatting will happen for example when you run:
 
-```
+```shell
 % mvn test
 ```
 
 You can manually run _only_ the formatting process with:
 
-```
+```shell
 % mvn prettier:write
-
 ```
 
-To skip the prettier formating use profile `prettierSkip`:
+To skip the Prettier formating, use the profile `prettierSkip`:
 
-```
+```shell
 % mvn test -P prettierSkip
 ```
 
-The check for formatting errors use profile `prettierCheck`:
+To check for formatting errors, use the profile `prettierCheck`:
 
-```
+```shell
 % mvn test -P prettierCheck
 ```
 
 The check is run by the CI server and will fail the build if the code is incorrectly formatted.
 
-### IntellJ and Code Style Formatting
+### IntelliJ and Code Style Formatting
 
-You should use the prettier Maven plugin to reformat the code or run prettier with Node(faster).
+You should use the Prettier Maven plugin to reformat the code or run Prettier with Node (faster).
 
-The prettier does NOT format the doc and markdown files, only Java code. So, for other files you should
-use the _project_ code-style. It is automatically imported when you first open the project. But, if
-you have set a custom code-style in your settings (as we used until OTP v2.1), then you need to
-change to the _Project_ Code Style. Open the `Preferences` from the menu and select _
-Editor > Code Style_. Then select **Project** in the \_Scheme drop down.
+Prettier does _not_ format the doc and Markdown files—only Java code. So, for other files, you
```

Comment:
Again, even though this is technically an em-dash character, it renders very similarly to a hyphen. Even for me as a native English speaker, the updated version confused me for a minute because I didn't understand what appeared to be a hyphenated word "files-only" modifying "Java code". It might be even more confusing for an international audience. This one makes me more seriously consider just never using em-dashes in this technical documentation.

Some of the added punctuation such as commas are probably a matter of style or taste. I tend to over-comma my prose so it generally doesn't bother me, but I'm not sure it's necessary here.

Reply:
Removed the em dash. For the added comma, it's a matter of discretion for commas after a conjunction at the beginning of the sentence ("So,") and after short prepositional phrases ("for other files,"). I'd tried to make the comma usages match, but it really could go either way so happy to revert.